upgrade-instructions.md

```diff
@@ -33,6 +33,17 @@ Change list:
  - [ProtocolSession storng typing](#protocolsession-storng-typing)
  - [Tune Cassandra time serie tables options](#tune-cassandra-time-serie-tables-options)
  - [Log4J2 Adoption](#log4j2-adoption)
+ - [Drop Cassandra schema version prior version 5](#drop-cassandra-schema-version-prior-version-5)
+
+### Drop Cassandra schema version prior version 5
+
+Date 06/04/2020
+
+JIRA: https://issues.apache.org/jira/browse/JAMES-2997
+
+In an effort to simplify the code base, we dropped support for Cassandra schema version prior version 5.
+
+Installation running older schema version needs to rely on release 3.5.0 to upgrade to schema version 7.
```

Comment:
maybe you can add that if you want to upgrade an older version you need to go through this and that version, no?

Reply:
Isn't it what I said here?

Reply:
oh yes, in fact it's explained.
As it was a little hard for me, let's rewrite it:
```suggestion
Installation running older schema version than version 5 needs to rely on release 3.5.0 to upgrade to schema version 7, before upgrading to an eventual newer version.
```